x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -7915,4 +7915,4 @@
     "xpack.watcher.watchActionsTitle": "满足后将执行 {watchActionsCount, plural, one{# 个操作} other {# 个操作}}",
     "xpack.watcher.watcherDescription": "通过创建、管理和监测警报来检测数据中的更改。"
   }
-}
\ No newline at end of file
+}
```

Comment:
@Avinar-24  please exclude this file from diff

Reply:
Linters don't let me do it.